templates/message/view/message-report-link.php

```diff
@@ -0,0 +1,7 @@
+<?php
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+?>
+<a href="#<?php if ( is_user_logged_in() ) : ?>message_report<?php else : ?>user_login<?php endif; ?>_modal" class="hp-listing__action hp-listing__action--report hp-link">
```

Comment:
It's ok to remove the condition because the template where this link is added will not be visible to logged out users for sure.